content/zh/docs/reference/setup-tools/kubeadm/kubeadm-reset-phase.md

```diff
@@ -91,7 +93,11 @@ Using this phase you can perform cleanup on this node.
 * [kubeadm reset](/docs/reference/setup-tools/kubeadm/kubeadm-reset/) to revert any changes made to this host by `kubeadm init` or `kubeadm join`
 * [kubeadm alpha](/docs/reference/setup-tools/kubeadm/kubeadm-alpha/) to try experimental functionality
 -->
-* [kubeadm init](/docs/reference/setup-tools/kubeadm/kubeadm-init/) 引导 Kubernetes 控制平面节点
-* [kubeadm join](/docs/reference/setup-tools/kubeadm/kubeadm-join/) 将节点连接到集群
-* [kubeadm reset](/docs/reference/setup-tools/kubeadm/kubeadm-reset/) 恢复通过 `kubeadm init` 或 `kubeadm join` 操作对主机所做的任何更改
-* [kubeadm alpha](/docs/reference/setup-tools/kubeadm/kubeadm-alpha/) 尝试实验性功能
+* [kubeadm init](/zh/docs/reference/setup-tools/kubeadm/kubeadm-init/)
+  引导 Kubernetes 控制平面节点
```

Comment:
是不是有点过度换行了，120字符以内不需要换行吧

Reply:
规范的英文文档都是保持在120内，超过120才换行

Reply:
中文换行整体上不影响段落，但会导致上下两行之间显示上多一个空格，就像这个：

![image](https://user-images.githubusercontent.com/9354727/101275518-39b0f600-37e1-11eb-93f5-729f839fea72.png)

![image](https://user-images.githubusercontent.com/9354727/101275549-6fee7580-37e1-11eb-92fd-0baf52968293.png)


个人建议禁用 IDE 的自动换行功能，自动换行过于武断，不利于控制排版。

Reply:
Because website will be frozen next Monday. 
This problem can be ignored for this pr now. 
If you have good suggestions, you can submit it to the [Chinese Localization Guidelines](https://kubernetes.io/zh/docs/contribute/localization_zh/) , thank you.